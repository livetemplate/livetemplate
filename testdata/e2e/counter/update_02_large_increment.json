{
  "2": "25",
  "5": "2023-01-01 10:10:00"
}
