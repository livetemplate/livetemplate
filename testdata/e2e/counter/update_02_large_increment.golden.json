{
  "1": "25",
  "4": "2023-01-01 10:10:00"
}